### ISSUE TITLE
Update on Warhammer: Vermintide 2 EAC

### ISSUE THREAD

**Issue by Lamdarer** [Starz0r/AreWeAntiCheatYet#637]
### Game Title

Warhammer: Vermintide 2

### Anti-Cheat Software

Easy Anti-Cheat

### Status of the Anti-Cheat

👍 Running

### Tinkering Steps

The game [got reported as working](https://steamcommunity.com/app/552500/discussions/0/2960544187643622408/?ctp=30#c3457094584911997892), though you might want to keep it as Broken until more ppl can validate if it reliable.
I will try it myself but for know it seem to require the following steps:
1. Beta: `temporary_launcher_workaround - Bypass Launcher`
2. Proton Experimental or GE-Proton7-28
3. Proton EAC Runtime installed in the Steam Library(tools)
EDIT: 4. It is possible that you have to be the host otherwise some users reported that they got kicked

Maybe some more ppl can test it before you change the Anti-Cheat Status of the game

### Proof of Mention

https://steamcommunity.com/app/552500/discussions/0/2960544187643622408/?ctp=31#c3457094584913365175

**Comment by Curve** [comment]
I'll keep an eye on it ^^

If more reports start rolling in I'll update the status

I'll also add an update for this game which describes the current situation, however I can only do that next week as I'm not home currently

**Comment by Lamdarer** [comment]
PS: I also got the game running and submitted my report to ProtonDB

1. Install the Game as well as the `Proton EasyAntiCheat Runtime` from within your Steam Library
2. Force Proton Experimental
3. Select the temporary_launcher_workaround - Bypass Launcher Beta
4. Delete `.../steamapps/compatdata/552500` (if it already exists on your PC from before)
5. Verify game integrity
6. Launch the Game, I had to replay the tutorial
7. Now you are in the game, but you have to host the lobby otherwise you will most likely get a timeout during a usual quick match (hopefully they can fix this)

**Comment by cecton** [comment]
I tried yesterday but got a warning on the screen about EAC.

I moved the installation from a Windows disk to a Linux partition. I selected Proton Experimental and temporary_launcher_workaround (we know it's a lie, it's never "temporary" :grin:), it installed itself `Proton EasyAntiCheat Runtime` tool when I ran the game.

To be sure I also re-deleted the compatdata and I re-verified the game installation. (Verifying the installation re-installs the compatdata directory.)

The error message says:

```
State: Untrusted
insufficient resources, cannot continue
something something your progress won't be saved
```

I can't make a screenshot unfortunately.

Maybe noteworthy: at the end I'm just getting a black screen despite the message indicating that I could still play the game but nothing will be saved. I do see the the loading icon for a little while but then it's black.

The error message makes me think that something is broken for it to work on my system but EAC could maybe work for this game.

OS: ArchLinux
Kernel: 6.0.11-arch1-1

**Comment by cecton** [comment]
@Lamdarer it was a "my system" issue. I found [that comment](https://steamcommunity.com/app/552500/discussions/0/2960544187643622408/?ctp=13#c3361398331727239592) in the thread you linked:

>>> Make sure you have Proton Easy Anit Cheat installed as well 
>> well it does that every time I verify files whitch i've done multiple times, without a clue again I am, 
> **No you have to manually install the runtime from Steam.** The "Proton Easy Anticheat Runtime". I have a feeling 99% of peoples problems revolve around not doing this. 

So I gave it a try. It was actually difficult to uninstall `Proton EasyAntiCheat Runtime`, I had to delete the file `appmanifest_1826330.acf` manually and the directory `common/Proton EasyAntiCheat Runtime`. I also deleted the compatdata for Vermintide.

Then when I restarted Steam I first installed `Proton EasyAntiCheat Runtime`. :warning: **I couldn't find HOW to install it manually** because the Steam Store does not list it. So, **before I uninstalled it**, I added it to the favorites in order to be able to select it in the list.

After installing this. I re-started the check file integrity. Then I started the game and everything seems to be working. I will give further updates here for others who might end up here.

**Comment by Curve** [comment]
> @Lamdarer it was a "my system" issue. I found [that comment](https://steamcommunity.com/app/552500/discussions/0/2960544187643622408/?ctp=13#c3361398331727239592) in the thread you linked:
> 
> >>> Make sure you have Proton Easy Anit Cheat installed as well 
> >>
> >> well it does that every time I verify files whitch i've done multiple times, without a clue again I am, 
> >
> > **No you have to manually install the runtime from Steam.** The "Proton Easy Anticheat Runtime". I have a feeling 99% of peoples problems revolve around not doing this. 
> 
> So I gave it a try. It was actually difficult to uninstall `Proton EasyAntiCheat Runtime`, I had to delete the file `appmanifest_1826330.acf` manually and the directory `common/Proton EasyAntiCheat Runtime`. I also deleted the compatdata for Vermintide.
> 
> Then when I restarted Steam I first installed `Proton EasyAntiCheat Runtime`. :warning: **I couldn't find HOW to install it manually** because the Steam Store does not list it. So, **before I uninstalled it**, I added it to the favorites in order to be able to select it in the list.
> 
> After installing this. I re-started the check file integrity. Then I started the game and everything seems to be working. I will give further updates here for others who might end up here.

It's not in the steam store but it can be found in your games list when you toggle the option to show "tools"

**Comment by cecton** [comment]
> It's not in the steam store but it can be found in your games list when you toggle the option to show "tools"

If it's uninstalled you won't see it no?

**Comment by Curve** [comment]
> > It's not in the steam store but it can be found in your games list when you toggle the option to show "tools"
> 
> If it's uninstalled you won't see it no?

You should still see it, at least I do

**Comment by cecton** [comment]
> You should still see it, at least I do

Yes I think you're right. I wrongly assumed I would not see it but I see everything, even servers for games I don't own. So I suppose there is a mix of everything in there.

> I will give further updates here for others who might end up here.

1. I copied the user setting directory located in Roaming. The mods I had are listed as enabled in the config file but they are not enabled in game. I think it won't enable without the launcher.
2. I tried a quick play. I got disconnected early after the game launched. I guess joining any game won't work.
3. Hosting game works perfectly.
4. I started the game about 7 times and there is 1 time it failed with the error "State: Untrusted". So it's not working consistently. Restarting the game right after without doing anything worked though.
